WORKSPACE

```diff
@@ -1,5 +1,16 @@
 workspace(name = "io_bazel_rules_scala")
 
+
+####### previous rules scala for building of scala compiler plugin
+rules_scala_version="3070353d06bb726141e6cea241e5d6e355a4d14f" # update this as needed
```

Comment:
removed http_archive of old rules_scala version.
added bootstrapping version of scala_library rule